webapp/javascript/components/Settings/Users/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect, useState } from 'react';
+import Button from '@ui/Button';
+import { useAppDispatch, useAppSelector } from '@pyroscope/redux/hooks';
+import { reloadUsers, selectUsers } from '@pyroscope/redux/reducers/settings';
+import UserTableItem from './UserTableItem';
+
+import userStyles from './Users.module.css';
+import tableStyles from '../SettingsTable.module.css';
+import UserAddForm from './UserAddForm';
+
+const sampleUsers = [
+  {
+    login: 'SampleUserLogin',
+    name: 'Test User Name',
+    email: 'sampleUserEmail@gmail.com',
+    lastActive: 'never',
+  },
+  {
+    login: 'shaleynikov',
+    name: 'Test User 2 Name',
+    email: 'sampleUser2Email@gmail.com',
+    lastActive: 'yesterday',
+  },
+];
+
+function Users() {
+  const dispatch = useAppDispatch();
+  const users = useAppSelector(selectUsers);
+  console.log('Rendering users');
+  useEffect(() => {
+    if (typeof users === 'undefined') {
+      console.log('Reloading users');
+      dispatch(reloadUsers());
+    }
+  });
```

Comment:
Is this the case where we load users once?
If so i think it can be written as 
```suggestion
  useEffect(() => {
      dispatch(reloadUsers());
  }, []);
```